<!-- ... -->
<launch>

  <include file="$(find ocams)/launch/ocams_ros.launch" />

  <node pkg="camera_calibration" type="cameracalibrator.py" name="camera_calibration" output="screen" args="--size 8x6 --square 0.03 --no-service-check true" >
    <remap from="left"   to="stereo/left/image_raw"/>
    <remap from="right"  to="stereo/right/image_raw"/>
  </node>

</launch>
